feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/specialfolders/SpecialFoldersViewModel.kt

```diff
@@ -129,23 +129,14 @@ class SpecialFoldersViewModel(
 
     private fun navigateNext() {
```

Comment:
By changing this to `fun navigateNext(isManualSetup: Boolean)` we can pass in the appropriate value at the call sites and don't need `State.isManualSetup`.